### DISCUSSION TITLE
Production application deployment

### DISCUSSION THREAD

**Discussion by Guorvo** [benvinegar/counterscale#70] [Q&A] [answered]
Amazing work!, I'm having some difficulties trying to host a webpage application using Siteground.
What files should i look for to upload, or is it even possible to host without the miniflare server?

**Comment by Guorvo** [comment]
> Amazing work!, I'm having some difficulties trying to host a webpage application using Siteground. What files should i look for to upload, or is it even possible to host without the miniflare server?

I followed all the steps, I'm stuck on the build files, can't seem to find a build HTML to reference all the scripts

**Comment by benvinegar** [comment]
I'm sorry – I don't know what Siteground is, or what you're trying to achieve with "host a webpage". Do you mean your own personal webpage?

Counterscale just deploys itself using the instructions provided. It's not a tool for hosting generic websites. It's just an analytics tracker and dashboard for your _existing_ sites. How you host those sites is outside the scope of this project.

**Reply by Guorvo** [reply]
My bad, i didn't explain correctly what I'm trying to do.
I want to add the dashboard to a webpage, and i don't know where the dashboard build files are located to host them

**Reply by benvinegar** [reply, marked as answer]
Unfortunately, Counterscale is designed only to run on Cloudflare using the provided instructions. 

You cannot just host the build files somewhere else; the server generates the dashboard HTML responses (w/ data) on demand. If you want the dashboard to "appear" on your website, you'd have to point a custom subdomain to point at the Cloudflare deployment.
